.github/scripts/auto-backport.py

```diff
@@ -192,21 +185,24 @@ def main():
         if not backport_labels:
             print(f'no backport label: {pr.number}')
             continue
-        if args.commits and not with_github_keyword_prefix(repo, pr):
+        if not with_github_keyword_prefix(repo, pr):
             continue
         if not repo.private and not scylladbbot_repo.has_in_collaborators(pr.user.login):
             logging.info(f"Sending an invite to {pr.user.login} to become a collaborator to {scylladbbot_repo.full_name} ")
             scylladbbot_repo.add_to_collaborators(pr.user.login)
             comment = f':warning:  @{pr.user.login} you have been added as collaborator to scylladbbot fork '
-            comment += f'Please check your inbox and approve the invitation, once it is done, please add the backport labels again\n'
-            create_pr_comment_and_remove_label(pr, comment)
-            continue
+            comment += f'Please check your inbox and approve the invitation, otherwise you will not be able to edit PR branch when needed\n'
+            pr.add_to_labels("scylladbbot/backport_error")
```

Comment:
better off adding a comment explaining the protocol: 

When a pull request is pending for backport but its author is not yet a collaborator of "scylladbbot", we attach a "scylladbbot/backport_error" label to the PR. This prevents the workflow from proceeding with the backport process until the author has been granted proper permissions -- the author should remove the label manually to re-trigger the backport workflow.